packages/forms-web-app/src/controllers/examination/session/examination-session.js

```diff
@@ -50,6 +50,16 @@ const getExaminationSubmissionId = (session) => {
 
 const getExaminationEmailAddress = (session) => getExaminationSession(session).email;
 
+const getExaminationSubmissionRetryErrorCount = (session) => {
+	const examinationSession = getExaminationSession(session);
+	return examinationSession.retryErrorCount;
+};
+const setExaminationSubmissionRetryErrorCount = (session) => {
```

Comment:
Just a question around the naming as it seems to be doing 2 things, creating or incrementing it. Maybe a chance to split it out?

Reply:
I think it's a setter. Consistent with other places we set defaults.